src/components/navigation/share-modal.tsx

```diff
@@ -119,12 +120,23 @@ export function ShareModal(props: ShareModalProps) {
   )
 }
 
-// TODO: needs type declaration
-function buildButtons(list: any, copyToClipboard: any) {
-  // TODO: needs type declaration
-  return list.map((entry: any) => (
+interface EntryProps {
```

Comment:
Name suffix shouldn't be `Prop` since this isn't inteded for a React Component, right? So rather `Data` or `Payload` or something.